.github/ISSUE_TEMPLATE/BUG.md

```diff
@@ -0,0 +1,48 @@
+---
+name: 🐞 Bug Report
+about: Something went awry and you'd like to tell us about it.
+
+---
+
+<!--
+  ⚡️ katchow! We 💛 issues.
+
+  Please - do not - remove this template.
+  Please - do not - skip or remove parts of this template.
+  Or your issue may be closed.
+
+  👉🏽 Need help or tech support? Please don't open an issue!
+  Head to https://gitter.im/rollup/rollup or https://stackoverflow.com/questions/tagged/rollupjs
+
+  ❤️ Rollup? Please consider supporting our collective:
+  👉 https://opencollective.com/rollup/donate
+-->
+
+- Rollup Version (if applicable):
+- Operating System (or Browser):
+- Node Version:
+- NPM Version:
+
+### How Do We Reproduce?
+
+<!--
+  Issues without minimal reproductions will be closed! Please provide a repro by:
+  1. Using the REPL at https://rollupjs.org/repl, or
+  2. Provide a repository link (Read https://git.io/fNzHA for instructions)
```

Comment:
I hate repository links, as it always seems like a great way to put a backdoor onto a maintainers machine to me (`npm install` is executing).

```
2. Work to isolate the issue and write out the exact steps in the issue here.
3. Include a minimal GitHub repo link. These may take more time for us to look into than the above.
```

Reply:
I'm good with this change. For consensus @lukastaegert please weigh in on it (Lukas had previously asked for the REPL and repo link in order from the template on master).

Reply:
I think I am ok with @guybedford's change as it would still encourage the creation of a repo albeit with lower priority. From the debugging perspective, they can be a huge shortcut but I see the security concerns.

Maybe in the future we could work to extend the REPL to be able to reproduce more situations by adding a way to designate multiple entry points (maybe a simple checkbox above each module?) and some way (cogwheel button?) to provide a select set of config options that would make sense here.